src/V3Dead.cpp

```diff
@@ -279,6 +279,7 @@ class DeadVisitor final : public VNVisitor {
             } else {  // Track like any other statement
                 iterateAndNextNull(nodep->lhsp());
             }
+            iterateNull(nodep->timingControlp());
         }
     }
 
```

Comment:
This can get called before they're global I think? Either way, `timingControlp` can also be a delay, and removing this line causes `t_delay` to fail with a broken dtype on the delay.